### ISSUE TITLE
"Report date" should contain the timezone

### ISSUE THREAD

**Issue by job** [AbuseIO/AbuseIO#155]
Looking at report that come out of coloclue's abuse.io server, I see the timezone is missing:

Without timezone specification, the date is meaningless.

I recommend ISO 8601, example: `2016-10-15T13:09:16Z` is used.

```
Ticket #305: Report for IP address XXX (Open Netbios Server)
Category: Information message, we strongly advice this matter to be resolved
Report date: 2016-08-28 01:37
Report count: 1
Source: Shadowserver
Reply or help: https://ash.abuseio.coloclue.net/?id=305&token=XX
Report information:
  - mac_address: 00-00-00-00-00-00
  - workgroup: WORKGROUP
  - machine_name: SERVER
  - username: SERVER
  - Address: XXX
```

**Comment by kruisdraad** [comment]
All stamps withing AbuseIO should be UTC. problem is that even though we can set a timezone, the original sender might defer from the setting. Changing the format to show the timezone would simplify parsing for receivers. 

Updating this for 4.1 release

**Comment by job** [comment]
Ask upstream abuse information providers to signal in ISO 8601 format too

**Comment by kruisdraad** [comment]
Currently available in 4.0 and up:

Ticket created 2016-12-29T09:06:51+00:00
Ticket modified 2016-12-29T09:06:51+00:00

You can globally configure this in the config/app.php (copy to /config/$env/app.php and update that file, not the original)

from:

    'date_format' => 'd-m-Y',
    'time_format' => 'H:i:s P',

to:

    'date_format' => '',
    'time_format' => 'Y-m-d\TH:i:sP',

or if you dont mind the space

    'date_format' => 'Y-m-d',
    'time_format' => '\TH:i:sP',

this is functionally from within laravel and not something we can influence directly unless adding a lot of extra code.

and upstream providers and changes ... thats wishfull thinking.

**Comment by job** [comment]
@kruisdraad perhaps, if the timestamp is in UTC, make it like this:

`Ticket modified 2016-12-29T09:06:51Z` (note the added Z)

so, `'time_format' => 'Y-m-d\TH:i:sP\Z',` ?

**Comment by kruisdraad** [comment]
that can be done, but its a setting you can make within the config. But you wont be able to do so in 3.0 as there is nothing centrally management time like laravel in 4.x and up.

from the defaults we wont update it at this time as the preference is different for everyone and using the laravel defaults makes upgrades to newer framework easier.

**Comment by job** [comment]
Not having UTC as the default is probably the most painful option from an
international cooperation perspective.

**Comment by kruisdraad** [comment]
has been implemented into 4.0

**Comment by job** [comment]
Was fixed in https://github.com/AbuseIO/AbuseIO/commit/2e157a337b3569a5ead8dfa3fb556eed89a07097
